project_one/index.html

```diff
@@ -0,0 +1,82 @@
+
+<html>
+<head>
+ <title> Soul BarberShop  </title> 
+ <link href="style.css" rel="stylesheet" type="text/css">
+
+</head>
+
+<body>
+  <div >
+
+      <!-- <button id ="grading-element-one"  onclick="myFunction()">yes</button> -->
+
+        <div>
+            <!-- <script type ="text/javascript">
+             
+               let hairStyle =[{hairStyle1 : 'Low Fade'} ,{hairStyle1:'High fade'},{hairStyle1: 'Mid fade'},{hairStyle1: 'high Top Fade'},{hairStyle1:'Crew Cut'}];
+                
+                 const getNewArray =(item) =>{
+            
+                    const updateArray ="<ol>"+ [item.hairStyle1, item.hairStyle]+"</ol>";
+                
+                    return updateArray;
+            
+                        
+                    console.log(updateArray);
+            
+                }
+            
+                const barberNameHandler= () =>{
+            
+                    document.getElementById("Showlist").innerHTML = hairStyle.map(getNewArray);}
+            </script> -->
+
+            
+                
+              </ul>
+              
+              <p>Click the button if you would like to give a donnation</p>
+              
+              <button id ="grading-element-one"  onclick="myFunction()">yes </button>
+
+              <ul id="myList"></ul>
+              
+              <script  id="grading-element-one-consequence">
```

Comment:
`id="grading-element-one-consequence"` should NOT be present in the html. Javascript should put it into the DOM as part of `myFunction` being called. So add `id="grading-element-one-consequence"` to `node` or another element you create in `myFunction`.